metadata/en-US/full_description.txt

```diff
@@ -1,12 +1,13 @@
 Kotatsu is a free and open source manga reader for Android.<br>
-Main Features:
-<ul>
-<li>Online manga catalogues</li>
-<li>Search manga by name and genre</li>
-<li>Reading history</li>
-<li>Favourites organized by user-defined categories</li>
-<li>Downloading manga and reading it offline. Third-party CBZ archives also supported</li>
-<li>Tablet-optimized material design UI</li>
-<li>Standard and Webtoon-optimized reader</li>
-<li>Notifications about new chapters with updates feed</li>
- </ul>
+**Main Features:**
```

Comment:
This file must be an html, not a markdown:

https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/#html-descriptions

Reply:
not must but "**may** use HTML tags".  If you look at other projects `full_description.txt` they are text/markdown based some with a smattering of HTML for links.   Admittedly a small sample but yours is the first I've seen using HTML

e.g. 

- https://github.com/proninyaroslav/blink-comparison/blob/master/fastlane/metadata/android/en-US/full_description.txt
- https://github.com/QuantumBadger/RedReader/blob/master/fastlane/metadata/android/en-US/full_description.txt
- https://github.com/billthefarmer/gurgle/blob/main/fastlane/metadata/android/en-GB/full_description.txt